libs/movie/src/lib/movie/+state/movie.service.ts

```diff
@@ -124,4 +127,17 @@ export class MovieService extends CollectionService<MovieState> {
       map(movies => movies.sort((a, b) => a.title.international < b.title.international ? -1 : 1))
     );
   }
+
+  public async updateStatus(movie: Movie, status: StoreStatus) {
+    await this.update(movie.id, (movie) => ({
+      ...movie,
+      app: {
+        ...movie.app,
+        [this.app]: {
+          ...movie.app[this.app],
+          status: status,
+        },
+      },
+    }));
+  }
```

Comment:
```suggestion
  public updateStatus(movieId: string, status: StoreStatus) {
    return this.update(movieId, movie => ({
      ...movie,
      app: {
        ...movie.app,
        [this.app]: {
          ...movie.app[this.app],
          status,
        },
      },
    }));
  }
```

Don't use same name for 'movie' in function parameters and in the call back, this can be confusing

Reply:
please use this new function in `libs/movie/src/lib/movie/dashboard/actions/actions.component.ts` also

Reply:
ok I see this is already done in https://github.com/blockframes/blockframes/pull/8152/files ;)

Reply:
So i dont change a things ?

Reply:
no, no, do the changes please. I was talking about `libs/movie/src/lib/movie/dashboard/actions/actions.component.ts` but once this PR will be merged, everything will be ok for `https://github.com/blockframes/blockframes/pull/8152/files` also